include/vcpkg/build.h

```diff
@@ -128,7 +128,7 @@ namespace vcpkg::Build
         BUILT_IN,
         ARIA2,
     };
-    const std::string& to_string(DownloadTool tool);
+    constexpr StringLiteral to_string(DownloadTool tool);
```

Comment:
I think this is actually broken because it's trying to implement the to_string protocol which expects an actual string.

Reply:
Fixed using `StringLiteral::to_string()`